src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -9,9 +9,11 @@ import {
   UploadService } from '../../core/services/upload.service';
 
 const DISPLAYABLE_CONTENT = ['gif', 'jpeg', 'jpg', 'png'];
-const BYTES_PER_MB = 1000000;
-const MAX_UPLOAD_SIZE = 10 * BYTES_PER_MB; // 10MB
-const MAX_VIDEO_UPLOAD_SIZE = 5 * BYTES_PER_MB; // 5MB
+const BYTES_PER_MB = 1024 * 1024;
+const MAX_UPLOAD_SIZE = 11 * BYTES_PER_MB; // 11MB to allow 10.x MB
+const SHOWN_MAX_UPLOAD_SIZE_MB = 10;
+const MAX_VIDEO_UPLOAD_SIZE = 6 * BYTES_PER_MB; // 6MB to allow 5.x MB
+const SHOWN_MAX_VIDEO_UPLOAD_SIZE_MB = 5;
```

Comment:
Let's reduce the number of magic literals by setting
`MAX_SIZE = (SHOWN_MAX_SIZE + 1) * BYTES_PER_MB`

Reply:
That's a good idea, thanks! I was hesitant to do the inverse (`SHOWN_MAX_VIDEO_UPLOAD_SIZE_MB = MAX_SIZE - 1`) due to potential negative numbers/unintended behavior but this solves it, thanks!